lighthouse/tests/beacon_node.rs

```diff
@@ -308,6 +308,13 @@ fn network_import_all_attestations_flag() {
         .with_config(|config| assert!(config.network.import_all_attestations));
 }
 #[test]
+fn network_shutdown_after_sync_flag() {
```

Comment:
Perhaps it would be good add another check that does the inverse of this, to prevent us from enabling by default.